packages/demo-app-ts/src/utils/useTopologyOptions.tsx

```diff
@@ -74,115 +67,149 @@ export const useTopologyOptions = (
         <label className="pf-v5-u-display-inline-block pf-v5-u-mr-md pf-v5-u-mt-sm">Layout</label>
       </SplitItem>
       <SplitItem>
-        <DropdownDeprecated
-          position={DropdownPositionDeprecated.right}
-          toggle={<DropdownToggleDeprecated onToggle={() => setLayoutDropdownOpen(!layoutDropdownOpen)}>{layout}</DropdownToggleDeprecated>}
+        <Dropdown
+          toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+            <MenuToggle ref={toggleRef} onClick={() => setLayoutDropdownOpen(!layoutDropdownOpen)}>
+              {layout}
+            </MenuToggle>
+          )}
           isOpen={layoutDropdownOpen}
```

Comment:
```suggestion
          isOpen={layoutDropdownOpen}
          onOpenChange={(isOpen) => setLayoutDropdownOpen(isOpen)}
```

Reply:
Thank you @jeff-phillips-18 , nice catch. Updated